perception/docker_tf/transfer_learning/labelbox_data_processing/labelbox2pascal/__init__.py

```diff
@@ -0,0 +1,135 @@
+import os
```

Comment:
We don't put code in `__init__`, we just use it to import from other files in a module